src/EventSubscriber/ValidateMonetizationEnabledSubscriber.php

```diff
@@ -67,7 +67,7 @@ public function __construct(MonetizationInterface $monetization, MessengerInterf
   public function validateMonetizationEnabled(GetResponseEvent $event) {
     /** @var \Symfony\Component\Routing\Route $current_route */
     if (($current_route = $event->getRequest()->get('_route_object')) && ($current_route->hasOption('_apigee_monetization_route'))) {
-      if (!$this->monetization->isMonetizationEnabled()) {
+      if (!$this->monetization->isMonetizationEnabled() && !$this->monetization->isOrganizationApigeeX()) {
```

Comment:
This logic is saying that all Apigee X orgs are able to be monetized without any other setting, have we confirmed this is true?